src/site/content/en/media/accessibility/index.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: Accessibility with the <track> tag
+authors:
+  - joemedley
+description: |
+  Accessibility isn't like icing on a cake. It's never anything you put on a
+  backlog with the hope of introducing later. Captions and screen reader
+  descriptions are the only way many users can experience your videos, and in
+  some jurisdictions, they're even required by law or regulation.
+date: 2020-08-20
+updated: 2020-08-20
+---
+
+Accessibility isn't like icing on a cake. It's never anything you put on a
+backlog with the hope of introducing it later. Captions and screen reader
+descriptions are the only way many users can experience your videos, and in some
+jurisdictions, they're even required by law or regulation.
+
+To add captions or screen reader descriptions to a web video, add a
+[`<track>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track) tag
+to a `<video>` tag. In addition to captions and screen reader descriptions, tags
+may also be used for subtitles and chapter titles. The can also help search
+engines understand what's in a video. Those capabilities are outside the scope
+fo this article.
+
+<figure class="w-figure  w-figure--inline-right">
```

Comment:
```suggestion
<figure class="w-figure">
```
This looks nice but it's causing the code sample to get truncated

![image](https://user-images.githubusercontent.com/4713486/91112289-8c521200-e637-11ea-9cc7-7f029b37dc76.png)